CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 4.4 (unreleased)
```

Comment:
Because Python 3.4 is still in major Linux distributions the only supported Python 3 version, dropping its support should result in a major version number increase, aka 5.0.
(Here and in setup.py)

Reply:
Thank you for the review.

Just pushed the requested update.